utils/_decorators.py

```diff
@@ -111,43 +112,32 @@ def _should_skip(condition=None, library=None, weblog_variant=None):
     return True
 
 
-def missing_feature(condition: bool = None, library=None, weblog_variant=None, reason=None):
-    """decorator, allow to mark a test function/class as missing"""
-
-    skip = _should_skip(library=library, weblog_variant=weblog_variant, condition=condition)
+def decorator(skip, condition, decorator_type, reason, ensure_jira, callback, function_or_class):
+    if inspect.isclass(function_or_class):
+        assert condition is not None, _MANIFEST_ERROR_MESSAGE
 
-    def decorator(function_or_class):
+    if ensure_jira:
```

Comment:
You can save `ensure_jira` argument, and it'll make the code clear that JIRA tickets are required for `bug`/`flaky`

```suggestion
    if decorator_type in ("bug", "flaky"):
```